common-schema.json

```diff
@@ -37,6 +37,21 @@
         },
         "dataProvider": {
           "type": "string"
+        },
+        "owner": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "oneOf": [ 
+              {
+                "format": "uri"
+              },
+              {
+                "minLength": 1,
+                "maxLength": 256,
+                "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$"
```

Comment:
instead of repeating here the pattern you could just cross-reference it

 "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/EntityIdentifierType"

Reply:
indeed!